README.md

```diff
@@ -197,6 +208,7 @@ API
 |`smoke_assert_body <string>`     | assert that the body contains `<string>`             |
 |`smoke_assert_code <code>`       | assert that there was a `<code>` response code       |
 |`smoke_assert_code_ok`           | assert that there was a `2xx` response code          |
+|`smoke_assert_headers <string>`  | assert that the headers contain `<string>`           |
```

Comment:
Another nice assert might be:

```
smoke_assert_header <key> <value>
```

Where `value` is optional. If the header is found, but the value is different we'll be able to tell you. (E.g. `Content-Type` is there, but it does not contain `text/html`.)